framework/SwiftPlot/BarChart.swift

```diff
@@ -505,12 +506,16 @@ extension BarGraph {
     }
 
     func drawTitle(renderer: Renderer) {
-        renderer.drawText(text: plotTitle.title, location: plotTitle.titleLocation, textSize: plotTitle.titleSize, strokeWidth: 1.2, angle: 0, isOriginShifted: false)
+        if (plotTitle != nil) {
```

Comment:
Generally, I try to avoid force-unwrapping where I can, so I tend to use let binding in cases like this. If you use the following:

```
guard let plotTitle = self.plotTitle else { return }
```

followed by your drawing code, you can avoid a level of indentation and force-unwrapping with !. Same with drawLabels and others below.